mllib/src/main/scala/org/apache/spark/mllib/classification/NaiveBayes.scala

```diff
@@ -19,6 +19,8 @@ package org.apache.spark.mllib.classification
 
 import java.lang.{Iterable => JIterable}
 
+import org.apache.spark.broadcast.Broadcast
```

Comment:
So import ordering, this should probably be moved down to be with the other org.apache.spark imports.